frameworks/Java/rapidoid/benchmark_config.json

```diff
@@ -1,24 +1,26 @@
 {
   "framework": "rapidoid",
-  "tests": [{
-    "default": {
-      "setup_file": "setup",
-      "json_url": "/json",
-      "plaintext_url": "/plaintext",
-      "port": 8080,
-      "approach": "Stripped",
-      "classification": "Platform",
-      "database": "None",
-      "framework": "rapidoid",
-      "language": "Java",
-      "orm": "Raw",
-      "platform": "Rapidoid",
-      "webserver": "None",
-      "os": "Linux",
-      "database_os": "Linux",
-      "display_name": "rapidoid",
-      "notes": "",
-      "versus": "rapidoid"
+  "tests": [
+    {
+      "http-fast": {
```

Comment:
could you change this back to 'default'? The way the suite is built,  each `benchmark_config.json` must contain at least a 'default' test.